src/c++/perf_analyzer/load_manager.cc

```diff
@@ -200,21 +129,26 @@ LoadManager::LoadManager(
   on_sequence_model_ =
       ((parser_->SchedulerType() == ModelParser::SEQUENCE) ||
        (parser_->SchedulerType() == ModelParser::ENSEMBLE_SEQUENCE));
+
+  data_loader_.reset(new DataLoader(batch_size_));
+
+  memory_manager_ = std::make_shared<pa::MemoryManager>(
```

Comment:
should this be created outside and passed in? So that ideally all this other crap doesn't need to be passed in anymore? And also so that we can easily mock it out in testing?

Reply:
I filed a story to look into data loader usage. We can revisit this then